### DISCUSSION TITLE
Welcome to sof Discussions!

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#6663] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iuliana-prodan** [comment]
Hi everybody!

I'm Iulia, part of the NXP's Audio team. We are enabling Audio Subsystem (and SOF ) on i.MX.
I've been away for one year, because I gave birth to a baby girl - Iris. 😊
After a maternity leave, I'm very glad to be back, learn new things, (virtually) meet new people and face new challenges.

**Reply by lgirdwood** [reply]
Congratulations @iuliana-prodan and welcome back !

**Reply by ranj063** [reply]
Congratulations @iuliana-prodan!
